app/com/gu/memsub/subsv2/services/CatalogService.scala

```diff
@@ -76,12 +76,14 @@ class CatalogService[M[_] : Monad](productIds: ProductIds, fetchCatalog: M[Strin
       ) (DigipackPlans)
     supporterPlus <- (
       one[SupporterPlus[Month.type]](plans, "Supporter Plus month", FrontendId.Monthly) |@|
-        one[SupporterPlus[Year.type]](plans, "Supporter Plus year", FrontendId.Yearly) |@|
-        one[SupporterPlus[Month.type]](plans, "Guardian Weekly Rest Of World Monthly", FrontendId.ThirdTierMonthlyROW) |@|
-        one[SupporterPlus[Year.type]](plans, "Guardian Weekly Rest Of World Annual", FrontendId.ThirdTierAnnualROW) |@|
-        one[SupporterPlus[Year.type]](plans, "Guardian Weekly Domestic Annual", FrontendId.ThirdTierAnnualDomestic) |@|
-        one[SupporterPlus[Month.type]](plans, "Guardian Weekly Domestic Monthly", FrontendId.ThirdTierMonthlyDomestic)
+        one[SupporterPlus[Year.type]](plans, "Supporter Plus year", FrontendId.Yearly)
       ) (SupporterPlusPlans)
+    tierThree <- (
+      one[TierThree[Month.type]](plans, "Supporter Plus & Guardian Weekly Domestic - Monthly", FrontendId.ThirdTierMonthlyDomestic) |@|
+        one[TierThree[Year.type]](plans, "Supporter Plus & Guardian Weekly ROW - Annual", FrontendId.ThirdTierAnnualDomestic) |@|
+        one[TierThree[Month.type]](plans, "Supporter Plus & Guardian Weekly ROW - Monthly", FrontendId.ThirdTierMonthlyROW) |@|
+        one[TierThree[Year.type]](plans, "Supporter Plus & Guardian Weekly Domestic - Annual", FrontendId.ThirdTierAnnualROW)
```

Comment:
these FrontendId__c should have changed
`ThirdTierAnnualDomestic` (before)
`TierThreeAnnualDomestic` (after)

Reply:
https://github.com/guardian/memsub-promotions/pull/361